docs/pvcscale.md

```diff
@@ -0,0 +1,179 @@
+$ time ansible-playbook -vv -i inventory workloads/pvcscale.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `true`
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `true`
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `true`
+Enables/disables running the workload pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`
+Location of kubeconfig on orchestration host. PVCScale tests supports only
+default at this time.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`
+Location of the ssh public key to authenticate to the pbench results server.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`
+Enables/disables the collection of pbench data on the pbench agent pods. These pods are deployed by the tooling playbook.
+
+### PBENCH_SERVER
+Default: There is no public default.
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+Default: There is no public default.
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+
+### PVCSCALE_TEST_PREFIX
+Default: `pvcscale`
+Test to prefix the pbench results.
+
+### PVCSCALE_CLEANUP
+Default: `true`
+Enables/disables cleanup of this workload on completion. Cleanup in
+PVCScale terminology means delete `pvcscale0` projects - this operation can be
+resource intensive in case there are many pods/pvcs in `pvcscale0` project created
+during test.
+
+### PVCSCALE_BASENAME
+Default: `pvcscale`
+Basename used by cluster loader for the project(s) it creates.
+
+### PVCSCALE_MAXPODS
+Default: `10`
+Maximum number of pods that will be created by the pvcscale workload. 
+Typically much higher values are used than the default for large scale tests.
+PVCScale does not mechanism to calculate maximum number of pods which it can
+start, so it is necessary to adapt `PVCSCALE_MAXPODS` to cluster capacity and
+execute it accordingly. Choosing `PVCSCALE_MAXPODS` to be high can lead that
```

Comment:
Choosing `PVCSCALE_MAXPODS` too high can result in too many pods/PVCs created, which can ... ?  Also consistency PVCs or pvcs (across this document).